File: Php/clase.php
<?php
// Definición de una clase llamada 'Persona'
class Persona {
    // Propiedades de la clase
    private $nombre;
    private $edad;

    // Constructor de la clase
    public function __construct($nombre, $edad) {
        $this->nombre = $nombre;
        $this->edad = $edad;
    }

    // Método para mostrar información de la persona
    public function mostrarInformacion() {
        echo "Nombre: " . $this->nombre . "\n";
        echo "Edad: " . $this->edad . " años\n";
    }
}

// Crear un objeto de la clase Persona
$persona1 = new Persona("Juan", 30);

// Llamar al método para mostrar información
$persona1->mostrarInformacion();
?>

File: C/clase.c
#include <iostream>
using namespace std;

// Definición de una clase llamada 'Persona'
class Persona {
public:
    // Constructor de la clase
    Persona(string nombre, int edad) {
        this->nombre = nombre;
        this->edad = edad;
    }

    // Método para mostrar información de la persona
    void mostrarInformacion() {
        cout << "Nombre: " << nombre << endl;
        cout << "Edad: " << edad << " años" << endl;
    }

private:
    string nombre;
    int edad;
};

int main() {
    // Crear un objeto de la clase Persona
    Persona persona1("Juan", 30);

    // Llamar al método para mostrar información
    persona1.mostrarInformacion();

    return 0;
}
